### ISSUE TITLE
npm i --save-dev geckodriver fails because no binary is installed

### ISSUE THREAD

**Issue by sdgandhi** [vladikoff/node-geckodriver#51]
# Repro
`npm i --save-dev geckodriver`

# Expected
geckodriver to be downloaded and installed

# Actual

```
pm ERR! path /Users/.../src/.../node_modules/geckodriver/bin/geckodriver
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/.../src/.../node_modules/geckodriver/bin/geckodriver'
npm ERR! enoent This is related to npm not being able to find a file.
```

# Notes
- The geckodriver folder under node_modules is never created before trying to `chmod`
- macOS Mojave
- npm -v 6.4.1

**Comment by vladikoff** [comment]
Should be fixed now in 1.41.1, sorry!

**Comment by sdgandhi** [comment]
@vladikoff confirmed!
